pkg/cli/interactive_tests/test_rotate_flush.tcl

```diff
@@ -1,24 +0,0 @@
-#! /usr/bin/env expect -f
```

Comment:
This file should remain and test the flush.

Reply:
I adjusted `flush_server_logs` to wait for the flush, so every caller of `flush_server_logs` implicitly tests the function. Seemed superfluous to have both given how expensive each of these tests is, but can add it back if you feel strongly!

Reply:
No it's ok. thanks